spec/features/user/show.html.erb_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
```

Comment:
- [ ] Here you need to have a test asserts you can see the number of posts the user has written and a test that asserts you can see the user's profile picture.